ui/src/timeMachine/components/Vis.tsx

```diff
@@ -133,8 +135,15 @@ const mstp = (state: AppState) => {
 
   const giraffeResult = getVisTable(state)
   const xColumn = getXColumnSelection(state)
-  const yColumn = getYColumnSelection(state)
-  const fillColumns = getFillColumnsSelection(state)
+
+  let yColumn, fillColumns
```

Comment:
style tip: just set the values here and avoid re-setting them in an `else` block. it'll save the following characters from having to be written:

`,`, `}`, `else`, `{`, `yColumn`, `fillColumns`, `}`

so it'll look like:
```ts
let yColumn = getYColumnSelection(state)
let fillColumns = getFillColumnsSelection(state)

if (activeTimeMachine.view.properties.type === 'mosaic') {
  yColumn = getMosaicYColumnSelection(state)
  fillColumns = getMosaicFillColumnsSelection(state)
}
```

again, just a tip. no way is right or wrong, but it's generally thought that the fewer bits of code you have to read and write, the fewer mistakes there'll be